packages/appkit/src/library/react/index.ts

```diff
@@ -141,16 +141,21 @@ export function useAppKitState() {
     throw new Error('Please call "createAppKit" before using "useAppKitState" hook')
   }
 
-  const [state, setState] = useState(modal.getState())
+  const [state, setState] = useState({ ...modal.getState(), initialized: false })
 
   useEffect(() => {
-    const unsubscribe = modal?.subscribeState(newState => {
-      setState({ ...newState })
-    })
-
-    return () => {
-      unsubscribe?.()
+    if (modal) {
```

Comment:
What if modal somehow wouldn't initialized?

Reply:
we already check above, if it's not we would've thrown!